odoo/addons/base/models/image_mixin.py

```diff
@@ -0,0 +1,109 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import models, fields, api, tools
+
+
+class ImageMixin(models.AbstractModel):
+    _name = 'image.mixin'
+    _description = "Image Mixin"
+
+    # all image fields are base64 encoded and PIL-supported
+
+    # all image_raw fields are technical and should not be displayed to the user
+    image_raw_original = fields.Binary("Raw Original Image", attachment=True)
+    image_raw_original_byte_size = fields.Integer("Byte Size of the Raw Original Image", compute='_compute_image_raw', store=True)
```

Comment:
do we need it ? 

duplicated of image_original_byte_size? no ?

Reply:
It's like all the other non-raw fields, they are duplicate when not using a fallback.

That said, in the backend since we use the non-raw field, we need this compute to be correct to update the image size in the kanban view.

Reply:
Actually yeah that's the issue, since the other has to be computed based on the image_original instead of image_raw_original, I guess we can remove it. It's just used in the backend view.